subsys/bluetooth/common/Kconfig

```diff
@@ -77,8 +77,19 @@ config BT_BUF_ACL_RX_SIZE
 	  In a Controller only build this will determine the maximum ACL size
 	  that the Controller will send to the Host.
 
+config BT_BUF_ACL_RX_COUNT_EXTRA
+	int "Number of extra incoming ACL data buffers"
+	default 0
+	range 0 65535
+	help
+	  Number of incoming extra ACL data buffers sent from the Controller to
+	  the Host.
+
+	  By default, the number of incoming ACL data buffers is equal to
+	  CONFIG_BT_MAX_CONN + 1.
+
 config BT_BUF_ACL_RX_COUNT
-	int "Number of incoming ACL data buffers"
+	int "[DEPRECATED] Number of incoming ACL data buffers"
```

Comment:
The way you've implemented this, deprecating is actually worse than simply removing the option. The reason is that the option is now silently ignored, i.e. the user might think their value is getting used, even though it's not. Either just remove the Kconfig option (so that the user gets a clear build error), or put some logic in place which still uses it in case it has been set by the app.

Reply:
You are right, my bad. I updated `BT_BUF_ACL_RX_COUNT` to get the maximum between `CONFIG_BT_MAX_CONN + 1` and `CONFIG_BT_BUF_ACL_RX_COUNT` until the deprecation is done.

Also, do you know if it's possible to select the `DEPRECATED` Kconfig symbol for something else than `bool` entry? I tried to select it here but it created errors.

Reply:
> Also, do you know if it's possible to select the `DEPRECATED` Kconfig symbol for something else than `bool` entry? I tried to select it here but it created errors.

I think the only way to do it would be indirectly from another Kconfig option with `select DEPRECATED if BT_BUF_ACL_RX_COUNT > 0`. Not if that's the recommended way in this case, or what the other option would be (e.g. do you add new one just for the sake of this or use some related but non-deprecated option). @carlescufi do you know?

Reply:
Yeah not sure adding a new one is a good option. But anyway, as long as it's marked deprecated in the documentation and in the release note/migration guide it should be fine.

Reply:
To deprecate a Kconfig option, you can do `select DEPRECATED` which will generate a deprecated warning when building